utils/spiel/m0_filesystem_stats

```diff
@@ -206,13 +206,13 @@ main()
 	fi
 
 	if [ -z "$profile_addr" ] ; then
-		profile_addr=$($consul_query_cmd | grep profiles: | sed -n -e 's/^.*\(profiles:\)/\1/p'  | awk '{print $1}'| sed 's/^[^:]*://' | sed 's/:/,/')
+		profile_addr=$($consul_query_cmd | grep profiles | sed -n -e 's/^.*\(profiles\)/\1/p'  | awk '{print $1}' | cut -d'/' -f2 | cut -d':' -f1,2 | sed 's/:/, /g')
 	fi
 
 	if [ -z "$client_addr" ] ; then
 		local cmd="tail -2"
 		[[  $systemd == 1 ]] && cmd="tail -1"
-		client_addr=$(get_endpoints_of m0_client_other | $cmd | head -1)
+		client_addr=$(get_endpoints_of m0_client_other | $cmd)
```

Comment:
Why is this changed?
If the last "head -1" is not needed, then, if the "tail -2", this will give 2 lines.
Please test it.